rxandroidble/src/main/java/com/polidea/rxandroidble/internal/operations/RxBleRadioOperationMtuRequest.java

```diff
@@ -0,0 +1,65 @@
+package com.polidea.rxandroidble.internal.operations;
+
+import android.bluetooth.BluetoothGatt;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.RequiresApi;
+
+import com.polidea.rxandroidble.exceptions.BleGattCannotStartException;
+import com.polidea.rxandroidble.exceptions.BleGattOperationType;
+import com.polidea.rxandroidble.internal.RxBleRadioOperation;
+import com.polidea.rxandroidble.internal.connection.RxBleGattCallback;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
```

Comment:
This import is unused according to https://travis-ci.org/Polidea/RxAndroidBle/builds/181702296 checkstyle.